index.html

```diff
@@ -2296,6 +2296,41 @@ <h1>Core Representations</h1>
 representations should be added here once that section has been written.
     </p>
 
+    <section>
+      <h2>Handling Properties</h2>
+      
+        <section>
+          <h2>Unknown Properties</h2>
+          <p>
+    When a DID Document is created by consuming a compliant representation, any properties
+    in the representation that are not known to the consumer MUST be added to the
+    data model as properties without any further processing on the property
+    value.
+          </p>
+          <p>
+    When a DID Document is serialized by a producer into a compliant
+    representation, any unknown properties MUST be serialized into the
+    representation. When the data type for the property is unknown by the producer,
+    a string serialization of the property SHOULD be used.
+          </p>
+        </section>
+  
+        <section>
+          <h2>Reserved Properties</h2>
+          <p>
+    When a DID Document is created by consuming a compliant representation, if any 
+    properties in the representation are reserved by another representation, as 
+    indicated in the registry entry for representations, then the consumer MUST
+    raise an error. 
+          </p>
+          <p>
+    When a DID Document is serialized by a producer into a compliant
+    representation, any reserved properties not for the current
+    representation MUST NOT be used.
```

Comment:
```suggestion
    representation MUST NOT be present in the serialized representation.
```

Reply:
I assume this means anywhere... in other words... no JSON-LD can be present ANYWHERE in a JSON DID representation... in other words... no `@context` nested inside a verificationMethod or service endpoint as well.... or does this only apply to the JSON path `$["@context"]` ?

Reply:
Yeah, this is highly problematic language... it means that you can't envelope any JSON-LD... so, right off of the bat you can't envelope JWTs that carry VCs (as a simple example to demonstrate why we didn't have this language in the spec before). Seemingly simple restrictions like this end up becoming highly problematic as an ecosystem develops.

Reply:
@OR13 No, it means _as a property_, which is a top-level element in the document. Properties can have as many of whichever sub-properties they want. 

I disagree with the doom and gloom from @msporny  -- now that we have a way to define a reserved property, this language actually limits the fall-on effects of keeping things out.

Reply:
if its just a top level property... doesn't that mean that the JSON document MUST allow nested JSON-LD?... 
Probably some additional language stating this would be necessary, as its not obvious today.... and then we're one logical step away from realizing that JSON-LD is JSON.... in the same way way that a `tiger` is a `cat`... we don't define all cats as `not tigers`... we should not be defining `JSON` as `NOT JSON-LD`.

Reply:
But you'd be a bit surprised if the pet store gave you a `tiger` when you asked for a `cat`, right? Your metaphor is accurate but not relevant here. I understand that JSON-LD is syntactically JSON, but what you're missing is that by dying on that hill you're declaring, effectively, that all JSON processors now need to deal with JSON-LD stuff and expect it to show up. We decided strongly against this at the Amsterdam F2F, and the representation text which this PR strengthens came from that discussion.

The JSON document doesn't need to allow anything that's not defined on a property. If a property is defined in terms of JSON-LD then it's up to the property to define how to process its value. This is very different from managing the document itself.

Reply:
would you be surprised to learn that Google and Microsoft return JSON-LD with content type `application/json`?

> The JSON document doesn't need to allow anything that's not defined on a property.

I agree.... JSON Documents do not need to allow anything.... but they do allow anything, for extensibility... thats why JSON is valuable.